### PR TITLE
Country of birth front

### PR DESCRIPTION
**Pull request by adamfaryna**
https://trello.com/c/W7XnNxhH

### CODE REVIEW

File: src/app/core/services/country.service.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    return { headers }
+  }
+
+  /*
+   * GET /api/country
+   */
+  getCountries(): Observable<CountryResponse[]> {
+    return this.http.get<any>("/api/country", this.getOptions())
+      .pipe(
+        debounceTime(500),
```
Comment (wozzer72) [inline src/app/core/services/country.service.ts]: Why is a debounce here? It's not as if this country request is initiated by username activity.

File: src/app/features/workers/country-of-birth/country-of-birth.component.html [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+            <input type="radio" formControlName="cobKnown" class="form-control" value="Other" for="cob_known_other">
+            <label for="cob_known_other">Other</label>
+          </div>
+        </div>
+      </div>
+
+      <div *ngIf="form.get('cobKnown').value === 'Other'" class="panel panel-border-narrow col-10 col-lg-6 ml-0 mt-5 mb-5 mt-md-0">
+        <div class="form-field">
+          <label class="form-label col-12 col-md-9">Country
+            <span class="example">Optional</span>
```
Comment (wozzer72) [inline src/app/features/workers/country-of-birth/country-of-birth.component.html]: "example" class?

File: src/app/core/services/country.service.ts (hunk 1 above)
Comment (adamfaryna) [reply src/app/core/services/country.service.ts]: Ok, I will remove it.

File: src/app/features/workers/country-of-birth/country-of-birth.component.html (hunk 2 above)
Comment (adamfaryna) [reply src/app/features/workers/country-of-birth/country-of-birth.component.html]: Yes, it's for displaying input examples.

Comment (wozzer72) [reply src/app/features/workers/country-of-birth/country-of-birth.component.html]: Thanks for the clarification.

File: src/app/core/services/country.service.ts (hunk 1 above)
Comment (wozzer72) [reply src/app/core/services/country.service.ts]: Cool.

**Pull request merged**